<launch>

  <arg name="BAG_LOCATION" value="/home/josef/Downloads/agbot_sensors_realsense_2019-12-03-16-53-49.bag"/> 

  <!-- Launch ROS Package Template Node -->
  <rosparam command="load" file="$(find agbot_kf)/config/default.yaml" />

  <!-- make the bag location a param -->
  <node pkg="rosbag" type="play" name="agbot_data" output="log" args="--loop -s 25 $(arg BAG_LOCATION)" />

  <node pkg="agbot_kf" type="agbot_kf" name="agbot_kf" launch-prefix="gdb -ex run -args" output="screen" />



</launch>